src/main/java/ProgramATM.java

```diff
@@ -21,16 +18,28 @@ public static void main(String[] args) {
         final String ANSI_BLACK_BACKGROUND = "\u001B[40m";
         final String BLUE_UNDERLINED = "\033[4;34m";
 
+
         try {
             Scanner scanner = new Scanner(System.in);
             if (tryAuthenticate(scanner)) {
                 CashMachine cashMachine = new CashMachine();
                 System.out.println(ANSI_GREEN + "Hello inside ATM bank");
 
                 do {
-                    payOut(scanner, cashMachine);
+                    System.out.print("how much money would you get?: ");
+                    int moneyQuantity = scanner.nextInt();
+
+                    int requestedSumInMain = cashMachine.payOut(moneyQuantity);
```

Comment:
To cashMachine powinno mi zwrócić wynik. Tak jak w prawdziwym bankomacie to maszyna zwraca Ci gotówkę.